charts/brigade/templates/api-role.yaml

```diff
@@ -1,15 +1,18 @@
 {{ if .Values.api.enabled }}
 {{ $fname := include "brigade.api.fullname" . }}
+{{ $serviceAccount := default $fname .Values.api.serviceAccount.name }}
+{{ if .Values.api.serviceAccount.create }}
```

Comment:
What do you think about a global `createServiceAccounts` bool (perhaps under the `rbac` section) instead of adding for each potential service account?  This assumes the use case is either all (can create service accounts) or none (cannot create any)...

Reply:
I did go that route at first, however looking at other helm charts in the main helm repo that have multiple service accounts, it seems that they typically have rbac and service account flags for each component.

Reply:
Ah, I see.  Indeed, I'd favor going with convention as well.